src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -128,8 +128,8 @@ class SentencingService(
                 )
                 courtCase.offenderCharges.add(offenderCharge)
                 courtCaseRepository.saveAndFlush(courtCase) // to access the newly created offender charges
-                courtEvent.initialiseCourtEventCharges()
               }
+              courtEvent.initialiseCourtEventCharges()
```

Comment:
this needed to run once all offender charges have been added - was causing a JPA merge issue.